k8s/aat/common/family-public-law/fpl-case-service.yaml

```diff
@@ -27,52 +27,11 @@ spec:
       image: hmctspublic.azurecr.io/fpl/case-service:prod-f4f18a39
       ingressHost: fpl-case-service-aat.service.core-compute-aat.internal
       environment:
-        IDAM_API_URL: https://idam-api.aat.platform.hmcts.net
-        IDAM_S2S_AUTH_URL: http://rpe-service-auth-provider-aat.service.core-compute-aat.internal
-        CORE_CASE_DATA_API_URL: http://ccd-data-store-api-aat.service.core-compute-aat.internal
-        DOCUMENT_MANAGEMENT_URL: http://dm-store-aat.service.core-compute-aat.internal
-        CCD_UI_BASE_URL: https://www-ccd.aat.platform.hmcts.net
-        DOCMOSIS_TORNADO_URL: https://docmosis-development.platform.hmcts.net
-        RD_PROFESSIONAL_API_URL: http://rd-professional-api-aat.service.core-compute-aat.internal
-        SEND_LETTER_URL: http://rpe-send-letter-service-aat.service.core-compute-aat.internal
-        FEES_REGISTER_API_URL: http://fees-register-api-aat.service.core-compute-aat.internal
-        PAYMENT_API_URL: http://payment-api-aat.service.core-compute-aat.internal
-        IDAM_CLIENT_ID: fpl_case_service
-        IDAM_CLIENT_REDIRECT_URI: https://fpl-case-service-*.service.core-compute-aat.internal/oauth2/callback
         GATEWAY_URL: https://gateway-ccd.aat.platform.hmcts.net
-        SPRING_MAIL_HOST: mta.reform.hmcts.net
-        SPRING_MAIL_PORT: 25
-        SPRING_MAIL_TEST_CONNECTION: false
```

Comment:
SPRING_MAIL_TEST_CONNECTION is not in chart

Reply:
By default this boolean property is false and have seen that this property is not overridden in the other ymls.

Reply:
you are right, default is false